assets/constants.ts

```diff
@@ -5175,6 +5175,7 @@ export = {
   },
 
   "ENABLE_PREREQUISITE_SKILLS": false,
+  "ENABLE_LITERALLYCANVAS_EDITOR": false,
```

Comment:
In the assets/literallyCanvas files below, there are a bunch of images. Where do these images come from and what is the licensing? You should probably include a README.

If they are from the literallyCanvas repo then we can't just stick them in our assets folder because that means we are changing the license. You may need to serve from third_party instead or create custom icons or use the existing fontawesome/material ones.

Reply:
I have added literallyCanvas in the third party library from which we can access the images. Done.